src/main/webapp/app/pages/curation/collapsible/MutationCollapsible.tsx

```diff
@@ -308,6 +315,25 @@ const MutationCollapsible = ({
         }
         isPendingDelete={isMutationPendingDelete}
       >
+        <RealtimeTextAreaInput
+          firebasePath={`${mutationPath}/summary`}
+          inputClass={styles.summaryTextarea}
+          label="Mutation Summary (Optional)"
+          labelIcon={
+            <GeneHistoryTooltip
+              historyData={parsedHistoryList}
+              location={`${getMutationName(mutationName, mutationAlterations)}, ${READABLE_FIELD.SUMMARY}`}
+              locationIdentifier={getLocationIdentifier({
+                mutationUuid,
+                fields: [READABLE_FIELD.SUMMARY],
+              })}
+            />
+          }
+          name="summary"
```

Comment:
Can we change to `mutationSummary` to prevent potential name clash with gene summary.